Core/PixelEvent.swift

```diff
@@ -776,6 +778,26 @@ extension Pixel {
         case settingsAccessibilityOpen
         case settingsAccessiblityTextZoom
 
+        case settingsPrivateSearchOpen
+        case settingsEmailProtectionOpen
+        case settingsEmailProtectionEnable
+        case settingsGeneralOpen
+        case settingsSyncOpen
+        case settingsAppearanceOpen
+        case settingsThemeSelectorPressed
+        case settingsAddressBarTopSelected
+        case settingsAddressBarBottomSelected
+        case settingsShowFullURLOn
+        case settingsShowFullURLOff
+        case settingsDataClearingOpen
+        case settingsFireButtonSelectorPressed
+        case settingsDataClearingClearDataOpen
+        case settingsAutomaticallyClearDataOn
```

Comment:
`settingsAutomaticallyClearDataOn` is fired for every action or timing parameter change in the Automatically Clear Data settings screen